content/en/project/status/2023MedleyAnnualReport.md

```diff
@@ -0,0 +1,91 @@
+* **UnixUtils:** In the last year we enhanced the set of tools that enables Medley to reach into its surrounding platform to accomplish tasks that are not available (and too extensive to implement) in Medley. As an example, the ShellBrowse function will open the specified URL within a browser running on the host platform. Similarly, ShellOpen can be used to open a PDF file using whatever PDF viewer is resident on the host platform.
+* **GITFNS (use of GIt and GitHub) extended**: Considerable effort went into the integration of Interlisp File Manager and GitHub, including a menu-driven interface to comparing Lisp source files on a function-by-function basis. This facility is crucial to the way the system manages the “residential style” development tools of Interlisp with GitHub.
+
+#### **Variety of Issues Closed and PRs Merged**
+
+The project development is managed through use of GitHub. There were 216 issues since the beginning of 2023, 69 of which closed, although many of those were website issues. Some of the issues led to 141 Pull Requests over the same time span, 132 of which merged. We continue to smooth some of the rough edges of using the Interlisp programming tools with Common Lisp compatible functions.
+
+#### **Community Engagement**
+
+We’ve actively engaged with the Lisp and Software Preservation communities, fostering collaboration and knowledge sharing.
+
+#### **Website Enhancements**
+
+The [Interlisp.org ](http://www.interlisp.org/)website continues to evolve. Our primary goal over the last year has been to clarify our messaging and evolve the website to support it. We have focused on adding content and redeveloping the site making content easier to find. In parallel we have worked to ensure our website is indexed allowing web searches on subjects related to Interlisp and Medley to direct users to appropriate content. Work on search optimization has also enhanced the internal website search capabilities. Efforts to improve the Interlisp bibliography continue across both the Zotero site holding the bibliographic material and within the Interlisp website.
+
+* **Content Organization and Accessibility:** In 2022, a first attempt at reorganizing the website was completed. Over the last year, we continued to revise and refine what and how to present material on our website. Changes are driven by new and improved content as well as improvements in our understanding of the story we need to tell. The result has been major improvements in website navigation. We continue to explore and work on new methods of enhancing our story telling. Work in progress includes a carousel on the landing page which will provide snapshots representing different facets of the Medley project.
```

Comment:
Is the carousel already active or in a branch waiting to be merged?

Reply:
I'm not sure. @phantomics?